src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const delimiters = ['-', '/', '.'];
+  const inputDelimiter = delimiters.find((d) => date.includes(d));
```

Comment:
Using `includes` to find the delimiter assumes that the delimiter will not be a part of the date parts. Ensure that the delimiter is not a number to avoid confusion with date parts.